### ISSUE TITLE
Change label for leisure=horse_riding

### ISSUE THREAD

**Issue by Hufkratzer** [openstreetmap/openstreetmap-website#3189]
Currently in en.yml and (en-GB.yml) the label for tag leisure=horse_riding is "Horse Riding":
<img width="183" alt="nearbyf" src="https://user-images.githubusercontent.com/5958184/116935468-b0560080-ac66-11eb-9c9c-cca6a66d8a36.png">

This is inappropriate since "horse riding" is an activity but the tag leisure=horse_riding stands for a physical place (like the other leisure tags usually do). The label should better be 
* "Horseback Riding Facility" in American English (en.yml) and 
* "Horse Riding Facility" in British English (so after changing in en.yml the string to "Horseback Riding Facility" this should be translated in Translatewiki to "Horse Riding Facility"and imported from there to en-GB.yml, right?)

Compare with
* leisure=dance -> label; "Dance Hall"
* leisure=fishing -> label: "Fishing Area"
* https://dictionary.cambridge.org/dictionary/english/horse-riding
* https://dictionary.cambridge.org/dictionary/english/horseback-riding

iD:
<img width="225" alt="iD" src="https://user-images.githubusercontent.com/5958184/116935171-463d5b80-ac66-11eb-9ca4-c9ea0cf679f3.png">

JOSM:
<img width="373" alt="JOSM" src="https://user-images.githubusercontent.com/5958184/116935213-548b7780-ac66-11eb-8647-b377d80b7077.png">

**Comment by tomhughes** [comment]
You realise that the JOSM menu is nothing to do with this repository right?

**Comment by Hufkratzer** [comment]
Yes, like iD and https://dictionary.cambridge.org; this was just for information.

**Comment by Hufkratzer** [comment]
The name for leisure=horse_riding was changed in iD; it's now:

| en | en-GB |
| ----- | ------- |
|  ![en](https://user-images.githubusercontent.com/5958184/171515488-c67f1a33-0088-4ebe-8987-13979f01bd9d.png) | ![en-GB](https://user-images.githubusercontent.com/5958184/171515642-33a0a920-656a-4e30-92d2-c37bf8a53a39.png) |

I propose to use that on the website too.

**Comment by Hufkratzer** [comment]
The name for leisure=horse_riding was also changed in JOSM (18646, 2023-01-30); it's now:
![josm-hrc](https://user-images.githubusercontent.com/5958184/218221457-3d7935dd-d30c-42d7-acf2-0f0ae63570a3.png)
(currently the same in **en** and **en_GB**)

Would it help if I created a PR to rename "Horse Riding" to "Horse Riding Centre" in en.yml?
